### PR TITLE
Update links and image for 1.9.0

### PR DESCRIPTION
**Pull request by misohu**
Related to: https://github.com/canonical/kubeflow-dashboard-operator/issues/190

Compared tags `v1.8.0` and `v1.9.0-rc.0` for files `https://github.com/kubeflow/kubeflow/tree/master/components/centraldashboard/manifests/overlays/kserve`

[Some of the links in the configmap ](https://github.com/kubeflow/kubeflow/blob/master/components/centraldashboard/manifests/base/configmap.yaml#L7)changed upstream. This PR resolves the changes related to this charm.

Changes:
- changing the order of some quick links related to upstream change

### CODE REVIEW

File: metadata.yaml [hunk 1]
```diff
@@ -12,7 +12,7 @@ resources:
     type: oci-image
     description: 'Backing OCI image'
     auto-fetch: true
-    upstream-source: docker.io/kubeflownotebookswg/centraldashboard:v1.8.0
+    upstream-source: docker.io/kubeflownotebookswg/centraldashboard:v1.9.0-rc.0
```
Comment (orfeas-k) [inline metadata.yaml:15]: regarding this, I didn't take a look at the manifests but I see this change in upstream image command. Should we update the charm's command as well? 
[1.8.0](https://github.com/kubeflow/kubeflow/blob/v1.8-branch/components/centraldashboard/Dockerfile) -> [1.9.0-rc.0](https://github.com/kubeflow/kubeflow/blob/v1.9.0-rc.0/components/centraldashboard/Dockerfile#L31)

Comment (NohaIhab) [reply metadata.yaml:15]: good point, updated
but if we will integrate the rock later then we will not use tini

Comment (DnPlas) [reply metadata.yaml:15]: I think that it is worth adding a comment just so we do not forget about changing this. Something like "remove tini when using the rock"

Comment (NohaIhab) [reply metadata.yaml:15]: added

Comment (orfeas-k) [reply metadata.yaml:15]: hmm what about adding the comment in the metadata.yaml? I feel like this is the place that guarantees we 'll see it when updating an image. I mean, we should in general take a look at the pebble layer and switching to rocks, but one could always forget about that.

Comment (orfeas-k) [inline metadata.yaml:15]: Hasn't upstream released a newer image than `rc.0`?

Comment (NohaIhab) [reply metadata.yaml:15]: looks like they [have](https://github.com/kubeflow/manifests/blob/master/apps/centraldashboard/upstream/base/kustomization.yaml#L16) but it's only in `master` not `1.9.0-rc.2`

Comment (NohaIhab) [reply metadata.yaml]: added a comment in metadata too

Comment (NohaIhab) [reply metadata.yaml]: I've updated it

**Review by DnPlas — APPROVED** [review]
LGTM, thanks @NohaIhab and @misohu !

**Pull request merged**